tests/features/kata.feature

```diff
@@ -32,6 +32,11 @@ Feature:
     When I evaluate the kata 'assert-true'
     Then I should see
     """
+    Star\Fixture\AssertTrue\AssertTrueObjective
```

Comment:
I suppose this is something new you added to the output with your refactor.. or was I running the command incorrectly?